src/globus_action_provider_tools/flask/helpers.py

```diff
@@ -74,7 +79,12 @@ def build_from_request(self, *, request: Request | None = None) -> AuthState:
         if not 10 <= len(access_token) <= 2048:
             raise UnverifiedAuthenticationError("Bearer token length is unexpected")
 
-        return super().build(access_token)
+        try:
+            return super().build(access_token)
+        except InvalidTokenScopesError as err:
+            raise AuthenticationError("token had invalid scopes") from err
+        except InactiveTokenError as err:
+            raise AuthenticationError("token was invalid, expired, or revoked") from err
```

Comment:
Past tense feels incorrect here.

Also, I prefer that our exception text follow sentence capitalization (I'm flexible on closing punctuation). All existing exceptions raised from this method are sentence-capitalized, so these additions clearly flag different authorship.

```suggestion
            raise AuthenticationError("Token has invalid scopes") from err
        except InactiveTokenError as err:
            raise AuthenticationError("Token is invalid, expired, or revoked") from err
```

Reply:
Yep, I also prefer title case. As evidenced here, I'm not perfectly consistent in my application of that rule.

As for past vs present... I can't say that I find either one better than the other, so I'm happy to apply the suggestion!
(I read the choices as the near equivalent: "It is invalid now, at the time of my evaluation" vs "It was invalid a moment ago, when I evaluated it")